app/javascript/mastodon/features/public_timeline/components/setting_text.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
Oh, sorry.
I thought about the possibility of individual change, but this function must be same in other timelines!!
I will move ```app/javascript/mastodon/features/home_timeline/components/setting_text.js``` to a new file ```app/javascript/mastodon/components/setting_text.js``` and import this from all the requiring files.
Is it OK?